about/release_policy.rst

```diff
@@ -3,48 +3,107 @@
+released and has received its first patch update. In practice, we support
+stable branches on a *best effort* basis for as long as they have active users
+who need maintenance updates.
+
+Whenever a new major version is released, we make the previous stable branch a
+long-term supported release, and do our best to provide fixes for issues
+encountered by users of that branch who cannot port complex projects to the new
+major version. This is the case for the 2.1 branch, and will be the case for
+the latest 3.x stable branch by the time Godot 4.0 is released.
+
+.. To add once 3.3 is released:
+   | Godot 3.4   | Q2 or Q3 2021     | |supported| *Beta.* Receives new features as well as bug fixes while     |
+   |             |                   | under development.                                                       |
 
 +-------------+-------------------+--------------------------------------------------------------------------+
 | **Version** | **Release date**  | **Support level**                                                        |
 +-------------+-------------------+--------------------------------------------------------------------------+
 | Godot 4.0   | ~2021 (see below) | |unstable| *Current focus of development (unstable).*                    |
 +-------------+-------------------+--------------------------------------------------------------------------+
-| Godot 3.2   | January 2020      | |supported| Backwards-compatible new features (backported from the       |
-|             |                   | ``master`` branch) as well as bug, security, and platform support fixes. |
+| Godot 3.3   | March/April 2021  | |supported| Receives bug, security and platform support fixes, as well   |
+|             |                   | as backwards-compatible usability enhancements.                          |
++-------------+-------------------+--------------------------------------------------------------------------+
+| Godot 3.2   | January 2020      | |partial| Only critical, security and platform support fixes             |
+|             |                   | (last update: 3.2.3).                                                    |
 +-------------+-------------------+--------------------------------------------------------------------------+
-| Godot 3.1   | March 2019        | |partial| Only critical, security and platform support fixes.            |
+| Godot 3.1   | March 2019        | |partial| Only critical, security and platform support fixes             |
+|             |                   | (last update: 3.1.2).                                                    |
 +-------------+-------------------+--------------------------------------------------------------------------+
-| Godot 3.0   | January 2018      | |partial| Only critical, security and platform support fixes.            |
+| Godot 3.0   | January 2018      | |eol| No longer supported (last update: 3.0.6).                          |
```

Comment:
Note: This makes 3.0 officially EOL. I haven't updated it in a year and I never hear from users who'd actually need a new release, so I think it's time to officially retire it.